core/primitives/Cargo.toml

```diff
@@ -6,11 +6,11 @@ edition = "2018"
 
 [dependencies]
 regex = "1"
-bincode = { version = "1.0", features = ["i128"] }
 bs58 = "0.3"
 base64 = "0.11"
 byteorder = "1.2"
 chrono = { version = "0.4.4", features = ["serde"] }
+derive_more = "0.99.3"
```

Comment:
It helped to clean up some duplicate code around `AsRef` / `Into` / etc common implementations.